### ISSUE TITLE
remove dependecy of curl replace with get_url module of ansible.

### ISSUE THREAD

**Issue by biholaindrasinh** [Vinelab/ansible-composer#3]

**Comment by jhughes-mc** [comment]
Tried to push a branch and pull request. I suppose I don't have permission. First time I've done this.

This should resolve your issue though. I updated the syntax while I was at it.

`---
- name: download composer
  get_url:
    url: https://getcomposer.org/installer
    dest: /tmp/installer
  tags: composer

- name: install composer
  shell: cat /tmp/installer | php -- --install-dir=/usr/local/bin
  args:
    creates: /usr/local/bin/composer
  tags: composer

- name: rename composer.phar to composer
  shell: mv /usr/local/bin/composer.phar /usr/local/bin/composer
  args:
    creates: /usr/local/bin/composer
  tags: composer

- name: make composer executable
  file: 
    path: /usr/local/bin/composer
    mode: a+x
    state: file
  tags: composer
`
